### ISSUE TITLE
[Broken Website] icon font

### ISSUE THREAD

**Issue by eqfae** [darkreader/darkreader#12459]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

class="twisty-container codicon-view-pane-container-expanded codicon"
github Codespaces Vscode web version
![图片](https://github.com/darkreader/darkreader/assets/120385812/b2dffb06-ddc4-4b0d-bdfc-7c6b3caf702f)
![图片](https://github.com/darkreader/darkreader/assets/120385812/ef00405f-72df-49f8-8c0b-21301da1df5c)

class="iconfont wp-s-aside-nav__sub-item-icon inline-block-v-middle icon-cate-pic"
![图片](https://github.com/darkreader/darkreader/assets/120385812/e904a7ea-f58a-4317-bf18-c7c26e29885c)

https://pan.baidu.com
![屏幕截图 2024-03-16 130218](https://github.com/darkreader/darkreader/assets/120385812/0e59809e-5896-41ac-add2-2f669536ff8b)
![屏幕截图 2024-03-16 130256](https://github.com/darkreader/darkreader/assets/120385812/1d3853b4-f071-4a11-8c16-1d5db91df4e1)

### Website Address

github.com pan.baidu.com 

### Steps To Reproduce

These font names all contain "icon"
Would it be possible to simply ignore substitutions containing icon fonts and then allow users to create exceptions?

### Operating System

win11

### Web Browser name and version

Firefox 115.8.0esr (20240212204114)

### Dark Reader version

4.9.78

**Comment by Myshor** [comment]
Yup... I confirm this is known issue when you force your own font.

I was making special sitefixes for that but it break if the the font is changed on site. Maybe @alexanderby will think how to exclude font changing for the classes including "icon" in name in the code.

**Comment by synthgab** [comment]
Hello,

Just a 2 cents thought on this topic, since I also tried to think of a solution 'cleaner' than manually maintaining a list of fonts to exclude: Unicode explicitly defines [Private Use Area](https://unicode.org/glossary/#private_use_area) , and if I'm not mistaken, most of these custom icon fonts are using characters in these blocks (mostly Exxx).

Would it be feasible to exclude the Private Use Area blocks from being overrided when a custom font is defined in Dark Reader ? I believe that, even if you had a custom font with glyphs in these blocks, by definition you would not want to override the one provided by the website unless you want icons to be completely messed up...
